README.md

```diff
@@ -20,4 +20,39 @@
 </div>
 
 ## gofuzz
-**gofuzz** is a CLI tool to add bug detection capabilities into your Go code. It transforms the source code and replace calls to functions/methods of interests by calls to corresponding hooks in the `github.com/CodeIntelligenceTesting/gofuzz/sanitizers` module. **gofuzz** does not change the code in-place, but generates the instrumented source code in a temporary directory and produces an [overlay file](https://go.dev/doc/go1.16) that can be used by your build tools
\ No newline at end of file
+**gofuzz** is a CLI tool to add bug detection capabilities into your Go code. It transforms the source code and replace calls to functions/methods of interests by calls to corresponding hooks in the `github.com/CodeIntelligenceTesting/gofuzz/sanitizers` module. **gofuzz** does not change the code in-place, but generates the instrumented source code in a temporary directory and produces an [overlay file](https://go.dev/doc/go1.16) that can be used by your build tools
+
+## How to use
+1. (Once) Install the **gofuzz** CLI
+    ```shell
+    go install github.com/CodeIntelligenceTesting/gofuzz/cmd/gofuzz@latest
+    ```
+2. (Once) Add the **sanitizers** package as a dependency for the code base you want to test. 
+This package contains the implementation of the hooks inserted by **gofuzz** into your code, 
+and therefore must be available when the instrumented code is being compiled.
+    ```shell
+    go get -u github.com/CodeIntelligenceTesting/gofuzz/sanitizers@latest 
+    ```
+3. Instrument your code using the **sanitize** subcommand
+   ```shell
+   gofuzz sanitize <package> -o <overlay.json>
+   ```
+   This instruments the specified package and write the instrumented file into a temporary
+   directory. The corresponding file replacements are stored in the <overlay.json> file.
+   By default, **gofuzz** writes a file named overlay.json in the current directory.
+4. Instrument your code for fuzzing using [go114-fuzz-build](https://github.com/kyakdan/go114-fuzz-build).
+   Note that we use a fork of the original [repo](https://github.com/mdempsky/go114-fuzz-build) 
+   because we need a change that has not yet been merged upstream. 
```

Comment:
As we do not know when or if the necessary patches are upstreamed anytime soon, we should add some clarification here as well. I propose:

- Add a `go install <your_fork> here, and/or
- Update the forks `README.md` and `go.mod` file accordingly as doing a `go install github.com/kyakdan/go114-fuzz-build` fails as your `go.mod` still points to the original repo by mdempsky.

Reply:
No idea. One PR is already here for a few months now. After cloning the repo, you can build the binary locally with `go build`, but this is not the standard Go way. I've just changed the package name on master so that `go install github.com/kyakdan/go114-fuzz-build` works